Sources/TuistCore/Log/File.swift

```diff
@@ -0,0 +1,2 @@
+import TuistSupport
+let logger = Logger(label: "io.tuist.core")
```

Comment:
Shouldn't the file be called `Logger.swift` instead of `File.swift`? I think it's more representative.